shared/src/store/action-reducers/exercise.ts

```diff
@@ -287,19 +287,18 @@ export function preparePartialExportForImport(
 }
 
 export interface TreatmentAssignment {
-    [patientId: UUID]: { [personnelType in PersonnelType]: number };
+    [patientId: UUID]: { [personnelType in string]: number };
```

Comment:
Use `ResourceDescription` instead of equivalent inline type (for the right hand side, i.e., `[patientId: UUID]: ResourceDescription`)